srcpkgs/docker/template

```diff
@@ -50,6 +51,8 @@ pre_build() {
 }
 
 do_install() {
+	vmkdir etc/docker
+	vinstall ${FILESDIR}/daemon.json 640 etc/docker daemon.json
```

Comment:
Use vconf instead

Reply:
I was unable to go this route. It is due to `vconf`'s requirement that the configuration exist at the depth of `/etc`; with no way of specifying additional directories beneath to the `vinstall` within the `vconf` function.